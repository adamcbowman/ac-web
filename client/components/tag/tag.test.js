import React from 'react'
import Renderer from 'react-test-renderer'
import { TagSet, Tag } from './'

test('tag component', () => {
    const tags = Renderer.create(
        <TagSet>
            <Tag>Tag #1</Tag>
            <Tag>Tag #2</Tag>
            <Tag>Tag #3</Tag>
        </TagSet>
    )

    expect(tags).toMatchSnapshot()
})
