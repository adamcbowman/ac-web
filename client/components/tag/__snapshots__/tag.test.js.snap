// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tag component 1`] = `
<ul
  className="Set"
>
  <li
    className="Item"
  >
    Tag #1
  </li>
  <li
    className="Item"
  >
    Tag #2
  </li>
  <li
    className="Item"
  >
    Tag #3
  </li>
</ul>
`;
